tools/lambda-promtail/lambda-promtail/s3.go

```diff
@@ -115,13 +115,16 @@ func getLabels(record events.S3EventRecord) (map[string]string, error) {
 
 func processS3Event(ctx context.Context, ev *events.S3Event) error {
 
+	fmt.Println("processing new S3Event\n")
+
 	batch, _ := newBatch(ctx)
 
 	for _, record := range ev.Records {
 		labels, err := getLabels(record)
 		if err != nil {
 			return err
 		}
+		fmt.Println("processing S3Record with key: %s bucket: %s", labels["key"], labels["bucket"])
```

Comment:
For formatting strings, you want `Printf`:
```suggestion
		fmt.Printf("processing S3Record with key: %s bucket: %s\n", labels["key"], labels["bucket"])
```